src/Microsoft.Health.Dicom.Core/Features/Store/StoreDatasetValidator.cs

```diff
@@ -250,15 +304,15 @@ string EnsureRequiredTagIsPresent(DicomTag dicomTag)
                     }
                     catch (DicomValidationException ex)
                     {
-                        validationResultBuilder.Add(ex, item.Tag, isCoreTag: RequiredCoreTags.Contains(item.Tag));
+                        validationResultBuilder.Add(ex, item.Tag, isCoreTag: false);
```

Comment:
Is the implication here that the query tags cannot include core tags?

Reply:
QueryTags can include since we validate the core tags in the previous step and I think it should still choose to check for isCoreTag